streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedTasks.java

```diff
@@ -233,11 +231,11 @@ private RuntimeException suspendTasks(final Collection<Task> tasks) {
     }
 
     private void closeZombieTask(final Task task) {
-        log.warn("{} Producer of task {} fenced; closing zombie task", logPrefix, task.id());
+        log.warn("Producer of task {} fenced; closing zombie task", task.id());
         try {
             task.close(false, true);
         } catch (final Exception e) {
-            log.warn("{} Failed to close zombie {} due to {}, ignore and proceed", taskTypeName, logPrefix, e);
+            log.warn("{} Failed to close zombie due to {}, ignore and proceed", taskTypeName, e);
```

Comment:
e.getMessage will be more accurate.

Reply:
because it was there before my changes so preferring to leave it as.